runtime/tests/checker/arrays_dictionaries_test.go

```diff
@@ -950,19 +937,6 @@ func TestCheckInvalidDictionarySubtyping(t *testing.T) {
 	assert.IsType(t, &sema.TypeMismatchError{}, errs[0])
 }
 
-func TestCheckInvalidArrayElements(t *testing.T) {
```

Comment:
Maybe don't remove this test case but turn it into a valid one

Reply:
Added a set of tests under `TestCheckArraySupertypeInference` that covers all cases. So removed this since it's a duplicate now.